tests/ballerina-unit-test/src/test/resources/test-src/connectors/h2/h2_actions_test.bal

```diff
@@ -255,6 +259,39 @@ function testInitWithInvalidDbOptions() returns (int[]) {
     return selectFunction(testDB);
 }
 
+function testReInitEndpoint() returns int {
+    endpoint h2:Client testDB {
+        path: "./target/H2Client/",
+        name: "TestDBH2",
+        username: "SA",
+        password: "",
+        poolOptions: { maximumPoolSize: 1 }
+    };
+
+    h2:ClientEndpointConfiguration config = {
+        path: "./target/H2Client/",
+        name: "TestDBH2",
+        username: "SA",
+        password: "",
+        poolOptions: { maximumPoolSize: 1 }
+    };
+
+    testDB.stop();
+
+    testDB.init(config);
```

Comment:
What if we don't call stop but directly call init again?

Reply:
Yes, we need to decide how to handle that scenario. I have created an issue to track this.
https://github.com/ballerina-platform/ballerina-lang/issues/10084

Reply:
Also, lets try to change this test case a bit. Lets init with different config values (for example - different db name) and then run the test case. This way we make sure that a new config is used with init.